src/test/app/wasm_fixtures/copyFixtures.py

```diff
@@ -86,7 +87,7 @@ def process_c(project_name):
     )
     build_cmd = (
         f"$CC --sysroot=$SYSROOT -I$SYSROOT/include/wasm32-wasi "
```

Comment:
> -I$SYSROOT/include/wasm32-wasi
This shouldn't be here. This mentioned in David's security document - "make sure wasi is disabled"